src/current/v23.1/backup.md

```diff
@@ -143,19 +143,28 @@ The `system.users` table stores your users and their passwords. To restore your
+
+A backup job, like any read, cannot export a range if the range contains an [unresolved intent](architecture/transaction-layer.html#resolving-write-intents). While you typically will want bulk, background jobs like `BACKUP` to have as little impact on your foreground traffic as possible, it's still important for backups to complete (which maintains your [recovery point objective (RPO)](https://en.wikipedia.org/wiki/Disaster_recovery#Recovery_Point_Objective)).
+
+Unlike a normal [read transaction](architecture/reads-and-writes-overview.html#read-scenario) that will block until any uncommitted writes it encounters are resolved, a backup job will be allotted a fixed amount of CPU time to read the required keys and values. Once the backup's read request has exhausted this time, the backup will resume once it has been allocated more CPU time. This process allows for other requests, such as foreground SQL traffic to continue, almost unaffected, because there is a cap on how much CPU a backup job will take.
+
+You can monitor your cluster's [admission control system](admission-control.html) on the [Overload dashboard](ui-overload-dashboard.html). To monitor your backup jobs, refer to the [Backup and Restore Monitoring](backup-and-restore-monitoring.html) page.
+
+For a more technical explanation of elastic CPU, refer to the [Rubber control theory on the Go scheduler](https://www.cockroachlabs.com/blog/rubbing-control-theory/) blog post.
 
 We recommend always starting backups with a specific [timestamp](timestamp.html) at least 10 seconds in the past. For example:
 
 ~~~ sql
-> BACKUP...AS OF SYSTEM TIME '-10s';
+BACKUP...AS OF SYSTEM TIME '-10s';
 ~~~
 
-This improves performance by decreasing the likelihood that the `BACKUP` will be [retried because it contends with other statements/transactions](transactions.html#transaction-retries). However, because `AS OF SYSTEM TIME` returns historical data, your reads might be stale. Taking backups with `AS OF SYSTEM TIME '-10s'` is a good best practice to reduce the number of still-running transactions you may encounter, since the backup will take priority and will force still-running transactions to restart after the backup is finished.
+This improves performance by decreasing the likelihood that the `BACKUP` will be [retried because it contends with other statements/transactions](transactions.html#transaction-retries). However, because [`AS OF SYSTEM TIME`](as-of-system-time.html) returns historical data, your reads might be stale. Taking backups with `AS OF SYSTEM TIME '-10s'` is a best practice to reduce the number of still-running transactions you may encounter, since the backup will take priority and will force still-running transactions to restart after the backup is finished.
```

Comment:
Is this clause still accurate: 

> since the backup will take priority and will force still-running transactions to restart after the backup is finished.

Reply:
This clause is still accurate but I'd suggest a rewrite:

Taking backups with `AS OF SYSTEM TIME '-10s'` is a good best practice to reduce the number of still-running transactions you may encounter, since contending transactions will be eventually pushed by the backup to a higher timestamp causing the transactions to retry.